yash-cli/tests/scripted_test/declutil-p.sh

```
@@ -0,0 +1,75 @@
+# declutil-p.sh: test of declaration utilities for any POSIX-compliant shell
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)